test/integration/01-setup/01-metadata.spec.ts

```diff
@@ -0,0 +1,7 @@
+import { DbMetadata } from "src/schema"
```

Comment:
this test is really to populating the database when doing manual request from the mobile app. the mobile app expect a `minBuildNumber` and `lastBuildNumber` to be returned from graphql.

maybe it shouldn't be there?

Reply:
a db "`hydratation`"

Reply:
uhmm I thought it was required for other tests.. if is not testing anything and if it is not a setup then we can remove it. do you agree?

Reply:
although if the same environment is used to run e2e tests in mobile app then we should leave it as it is

Reply:
there is no e2e test (yet) on the mobile side. 

I don't think we should just remove it but it doesn't need to be part of the integration test. it can be part of a script to be run before someone want to launch graphql to test the mobile app.

Reply:
Yeah some kind of
```
make prep-mobile-tests
```
that will bring everything up ready to run the mobile tests or something like that.

Reply:
uhmmm I moved to debug folder (according to the conversation with Nicolas) https://github.com/GaloyMoney/galoy/pull/340/commits/3353ed95c63c0204258a34c7bd891032c3717074 
Also, not sure what involves prep environment for mobile testing but we can add it then